plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -1280,7 +1281,7 @@ public ExecutionResult executeInVR(String routerIP, String script, String args,
         try {
             VmwareManager mgr = getServiceContext().getStockObject(VmwareManager.CONTEXT_STOCK_NAME);
             result = SshHelper.sshExecute(routerIP, DefaultDomRSshPort, "root", mgr.getSystemVMKeyFile(), null, "/opt/cloud/bin/" + script + " " + args,
-                    60000, 60000, timeout * 1000);
+                    60000, 60000, (int)timeout.getMillis());
```

Comment:
Please consider adding an overridden version of the `SshHelper.sshExecute(String, int, String, String, ??, String, int, int, int)` method that accepts `Duration` to encapsulate this type conversion.